packages/babel-plugin/src/index.js

```diff
@@ -0,0 +1,122 @@
+import {declare} from '@babel/helper-plugin-utils'
+import {create as createJss} from 'jss'
+
+const rawRuleName = '@raw'
+const defaultIdentifiers = ['createStyleSheet', 'injectSheet']
+
+export default declare(({types: t, ...api}, {identifiers = defaultIdentifiers, jssOptions}) => {
+  api.assertVersion(7)
```

Comment:
Should we maybe update our own babel packages to v7 as well? Would make testing easier.

Reply:
sure, if you get time, v6 and 7 are largely compatible